docs-gen/content/rule_set/basics.md

```diff
@@ -103,22 +103,37 @@ Vehicle.Navigation.CurrentLocation:
 
 It is recommended for servers, which are implementing protocols for the vehicle signal specification, to serve old and new nodes during the deprecation period described above.
 
+## Naming Rules
+
+Different languages and environments have different rules for identifiers.
+To simplify usage of VSS in as many languages and environments as possible the following rules apply.
+
+* The allowed character set for node names (branches and signals) is `A-Z`, `a-z` and `0-9`
```

Comment:
branches and signals -> branch and data nodes

Perhaps? To align with terms used elsewhere?

Reply:
Will change it. We are a bit inconsistent in documentation, like in https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/. We sometimes talks about signals, sometimes about data entries.